src/service/logs/kinesis_firehose.rs

```diff
@@ -325,12 +349,14 @@ pub async fn process(
     }))
 }
 
-fn decode_and_decompress(encoded_data: &str) -> Result<String, Box<dyn std::error::Error>> {
+fn decode_and_decompress(encoded_data: &str) -> Result<(String, bool), Box<dyn std::error::Error>> {
```

Comment:
Please document the returned `bool`. I don't understand it.

If you apply the suggestion, don't forget to fill the `[XXX-DOCUMENTME]` placeholder.
```suggestion
/// Decode `input` from Base64 representation and decompress the result into a `String`.
/// Returned `bool` signifies whether [XXX-DOCUMENTME].
fn decode_and_decompress(input: &str) -> Result<(String, bool), anyhow::Error> {
```